nautobot/core/settings.py

```diff
@@ -648,34 +647,14 @@
 # Caching
 #
 
-# v2 TODO(jathan): Remove all cacheops settings.
-# The django-cacheops plugin is used to cache querysets. The built-in Django
-# caching is not used.
-CACHEOPS = {
-    "auth.user": {"ops": "get", "timeout": 60 * 15},
-    "auth.*": {"ops": ("fetch", "get")},
-    "auth.permission": {"ops": "all"},
-    "circuits.*": {"ops": "all"},
-    "dcim.*": {"ops": "all"},
-    "ipam.*": {"ops": "all"},
-    "extras.*": {"ops": "all"},
-    "users.*": {"ops": "all"},
-    "tenancy.*": {"ops": "all"},
-    "virtualization.*": {"ops": "all"},
-}
-CACHEOPS_DEGRADE_ON_FAILURE = True
-CACHEOPS_ENABLED = is_truthy(os.getenv("NAUTOBOT_CACHEOPS_ENABLED", "False"))
-CACHEOPS_REDIS = os.getenv("NAUTOBOT_CACHEOPS_REDIS", parse_redis_connection(redis_database=1))
-CACHEOPS_DEFAULTS = {"timeout": int(os.getenv("NAUTOBOT_CACHEOPS_TIMEOUT", "900"))}
-
 # The django-redis cache is used to establish concurrent locks using Redis.
 CACHES = {
     "default": {
         "BACKEND": os.getenv(
             "NAUTOBOT_CACHES_BACKEND",
             "django_prometheus.cache.backends.redis.RedisCache" if METRICS_ENABLED else "django_redis.cache.RedisCache",
         ),
-        "LOCATION": parse_redis_connection(redis_database=0),
+        "LOCATION": parse_redis_connection(redis_database=1),
```

Comment:
I'm not entirely sure about the correctness and implications of this change. It "feels" right to me that CACHES should use a different redis DB than Celery though. Probably needs to be documented as a change at least. Tagging @nniehoff for some visibility.

Reply:
This makes sense to me, most of our deployments are explicit and we separate the database indexes... Also it's important to note that redis clusters (not sentinel) only support 1 DB, in which case it might be nice to use the same one but I think it's still cleaner to not